apps/backend/src/db/migrations/20240215141622_create_friends_table.ts

```diff
@@ -0,0 +1,46 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "friends";
+
+const ColumnName = {
+	CREATED_AT: "created_at",
+	ID: "id",
+	IS_INVITATION_ACCEPTED: "is_invitation_accepted",
+	RECIPIENT_USER_ID: "recipient_user_id",
+	SENDER_USER_ID: "sender_user_id",
+	UPDATED_AT: "updated_at",
+} as const;
+
+function up(knex: Knex): Promise<void> {
+	return knex.schema.createTable(TABLE_NAME, (table) => {
+		table.increments(ColumnName.ID).primary();
+		table
+			.dateTime(ColumnName.CREATED_AT)
+			.notNullable()
+			.defaultTo(knex.fn.now());
+		table
+			.dateTime(ColumnName.UPDATED_AT)
+			.notNullable()
+			.defaultTo(knex.fn.now());
+		table
+			.integer(ColumnName.SENDER_USER_ID)
+			.references("id")
+			.inTable("users")
+			.notNullable()
+			.onDelete("CASCADE");
```

Comment:
```suggestion
			.onDelete(DELETE_STRATEGY);
```
Create a constant for that